plugins/opds.koplugin/opdsbrowser.lua

```diff
@@ -695,7 +695,40 @@ function OPDSBrowser:downloadFile(item, filename, remote_url)
     end
 end
 
-function OPDSBrowser:streamPages(item, remote_url, count)
+-- This function shows a dialog with input fields
+-- for entering information for Page streaming from.
+-- an input page number
+function OPDSBrowser:jumpToPage(viewer)
+    local input_dialog
+    input_dialog = InputDialog:new{
+        title = _("Enter Page Number"),
```

Comment:
"Enter page number"

Reply:
Leaving a note for myself, this is done.

Reply:
GitHub may call it a suggestion, but this is not a suggestion. ;-)
```suggestion
        title = _("Enter page number"),
```

Reply:
This one still needs to be fixed.

Reply:
And `Enter page number`, not `Enter Page Number`. ;-)